### ISSUE TITLE
WebPanel can not created in another L&F

### ISSUE THREAD

**Issue by JFormDesigner** [mgarin/weblaf#382]
When creating a WebPanel from another L&F, I get a StyleException: StyleManager must be initialized.
Occurs in the current **styling** branch, but not in master.

Following program demonstrates the issue:
```java
import javax.swing.*;
import com.alee.laf.panel.WebPanel;

public class TestWebPanel
{
	public static void main( String[] args ) {
		SwingUtilities.invokeLater( new Runnable() {
			@Override
			public void run() {
				WebPanel panel = new WebPanel();
				panel.add( new JLabel("hello world") );

				JOptionPane.showMessageDialog( null, panel );
			}
		} );
	}
}
```

Throws following exceptions:
```
[AWT-EventQueue-0] ERROR com.alee.laf.panel.WebPanel - com.alee.managers.style.StyleException: StyleManager must be initialized
com.alee.managers.style.StyleException: StyleManager must be initialized
	at com.alee.managers.style.StyleManager.checkInitialization(StyleManager.java:175)
	at com.alee.managers.style.StyleManager.getData(StyleManager.java:686)
	at com.alee.managers.style.StyleManager.installSkin(StyleManager.java:428)
	at com.alee.laf.panel.WebPanelUI.installUI(WebPanelUI.java:80)
	at javax.swing.JComponent.setUI(JComponent.java:666)
	at javax.swing.JPanel.setUI(JPanel.java:153)
	at com.alee.laf.panel.WebPanel.updateUI(WebPanel.java:306)
	at javax.swing.JPanel.<init>(JPanel.java:86)
	at javax.swing.JPanel.<init>(JPanel.java:95)
	at com.alee.laf.panel.WebPanel.<init>(WebPanel.java:61)
	at TestWebPanel$1.run(TestWebPanel.java:10)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:756)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:76)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:726)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
Exception in thread "AWT-EventQueue-0" com.alee.managers.style.StyleException: StyleManager must be initialized
	at com.alee.managers.style.StyleManager.checkInitialization(StyleManager.java:175)
	at com.alee.managers.style.StyleManager.getData(StyleManager.java:686)
	at com.alee.managers.style.StyleManager.uninstallSkin(StyleManager.java:455)
	at com.alee.laf.panel.WebPanelUI.uninstallUI(WebPanelUI.java:92)
	at javax.swing.JComponent.uninstallUIAndProperties(JComponent.java:681)
	at javax.swing.JComponent.setUI(JComponent.java:658)
	at javax.swing.JPanel.setUI(JPanel.java:153)
	at com.alee.laf.panel.WebPanel.updateUI(WebPanel.java:311)
	at javax.swing.JPanel.<init>(JPanel.java:86)
	at javax.swing.JPanel.<init>(JPanel.java:95)
	at com.alee.laf.panel.WebPanel.<init>(WebPanel.java:61)
	at TestWebPanel$1.run(TestWebPanel.java:10)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:756)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:76)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:726)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

**Comment by mgarin** [comment]
This will be thrown in any `Web*` component if it tries to initialize before WebLaF managers are initialized. You can initialize managers by installing WebLaF as your application L&F or directly initializing all managers by calling `UIManagers.initialize ();`.

I am aware that this might not be possible to do with visual UI editors so I will look for a way to fix this behavior, probably by initializing managers on demand, but I'm not yet sure how that will affect L&F behavior so I can't push any quick workaround yet.

**Comment by JagFrank** [comment]
Any updates on this please?
I am getting this 
15:25:35.198][main] ←[1;31mERROR←[0;39m java.lang.Throwable - Caused by: com.alee.managers.style.StyleException: StyleManager must be initialized
15:25:35.199][main] ←[1;31mERROR←[0;39m java.lang.Throwable -  at com.alee.managers.style.StyleManager.checkInitialization(StyleManager.java:166)
15:25:35.199][main] ←[1;31mERROR←[0;39m java.lang.Throwable -  at com.alee.managers.style.StyleManager.getData(StyleManager.java:627)
15:25:35.199][main] ←[1;31mERROR←[0;39m java.lang.Throwable -  at com.alee.managers.style.StyleManager.installSkin(StyleManager.java:369)
15:25:35.199][main] ←[1;31mERROR←[0;39m java.lang.Throwable -  at com.alee.laf.panel.WebPanelUI.installUI(WebPanelUI.java:80)
15:25:35.200][main] ←[1;31mERROR←[0;39m java.lang.Throwable -  at javax.swing.JComponent.setUI(JComponent.java:664)
15:25:35.200][main] ←[1;31mERROR←[0;39m java.lang.Throwable -  at javax.swing.JPanel.setUI(JPanel.java:153)
15:25:35.200][main] ←[1;31mERROR←[0;39m java.lang.Throwable -  at javax.swing.JPanel.updateUI(JPanel.java:126)
15:25:35.200][main] ←[1;31mERROR←[0;39m java.lang.Throwable -  at javax.swing.JPanel.<init>(JPanel.java:86)
15:25:35.200][main] ←[1;31mERROR←[0;39m java.lang.Throwable -  at javax.swing.JPanel.<init>(JPanel.java:109)
15:25:35.200][main] ←[1;31mERROR←[0;39m java.lang.Throwable -  at javax.swing.JPanel.<init>(JPanel.java:117)
15:25:35.201][main] ←[1;31mERROR←[0;39m java.lang.Throwable -  at javax.swing.JRootPane.createGlassPane(JRootPane.java:545)
15:25:35.201][main] ←[1;31mERROR←[0;39m java.lang.Throwable -  at javax.swing.JRootPane.<init>(JRootPane.java:365)
15:25:35.201][main] ←[1;31mERROR←[0;39m java.lang.Throwable -  at javax.swing.JFrame.createRootPane(JFrame.java:277)
15:25:35.201][main] ←[1;31mERROR←[0;39m java.lang.Throwable -  at javax.swing.JFrame.frameInit(JFrame.java:258)
15:25:35.201][main] ←[1;31mERROR←[0;39m java.lang.Throwable -  at javax.swing.JFrame.<init>(JFrame.java:225)

**Comment by mgarin** [comment]
I haven't added any solution yet, but it is in the list of important things to be changed.
Most probably fix for this specific issue will be available closer to v1.2.9 release.

**Comment by mgarin** [comment]
This issue has the same cause as #18, shortly: you cannot initialize any WebLaF component without initializing WebLaF managers. I'll close this one, but you can still track issue #18 for the fix
